server/constants.py

```diff
@@ -33,7 +34,7 @@
 TIMEZONE = 'America/Los_Angeles'
 ISO_DATETIME_FMT = '%Y-%m-%d %H:%M:%S'
 
-AUTOGRADER_URL = 'https://autograder.cs61a.org'
+AUTOGRADER_URL = os.getenv('AUTOGRADER_URL', 'https://autograder.cs61a.org')
```

Comment:
I'd prefer if we move this out of constants and into the config if we're going to read from the env vars. I don't mind merging this as is, but if it's not a lot of work - we should move it to the config & change the (two afaik) places we use in the code.

Reply:
> Note that it's a bit of a misnomer to have the AUTOGRADER_URL in the constants file but dereference it from an environment variable. Likely a more consistent place would be the Flask app settings. However, the autograder is implemented outside of the Flask app context so

Nevermind, I just read this.